server.js

```diff
@@ -239,11 +241,31 @@ module.exports = function(options) {
   // This will catch all 404s and redirect them to root URL
   // with preserving the pathname for client-side to handle.
   server.ext('onPreResponse', function(request, reply) {
+    var url = request.url.pathname;
```

Comment:
Gragh this bothers the heck out of me. I wish we had a proper 404 page to serve up.